extension/docs/uddsketch.md

```diff
@@ -111,7 +112,59 @@ FROM daily_rain;
 (4 rows)
 ```
 
+## Example Using TimeScale Continuous Aggregates (uddsketch-cagg-example)
+To have a UddSketch over a PostgresQL table which automatically updates as more data is added, we can make use of continuous aggregates.  First, let us create a simple hypertable:
+
+```SQL ,non-transactional,ignore-output
+SET TIME ZONE 'UTC';
+CREATE TABLE test(time TIMESTAMPTZ, value DOUBLE PRECISION);
+SELECT create_hypertable('test', 'time');
+```
+
+Now we'll create a continuous aggregate which will group all the points for each week into a UddSketch:
+```SQL ,non-transactional,ignore-output
+CREATE MATERIALIZED VIEW weekly_sketch
+WITH (timescaledb.continuous)
+AS SELECT
+    time_bucket('1 week'::interval, time) as week,
+    timescale_analytics_experimental.uddsketch(100, 0.005, value)
+FROM test
+GROUP BY time_bucket('1 week'::interval, time);
+```
+
+Next we'll use one of our utility functions, `generate_periodic_normal_series`, to add some data to the table.  Using default arguments, this function will add 28 days of data points at 10 minute intervals.
+```SQL ,non-transactional
+INSERT INTO test
+    SELECT time, value
+    FROM timescale_analytics_experimental.generate_periodic_normal_series('2020-01-01 UTC'::timestamptz, NULL, NULL, NULL, NULL, NULL, NULL, rng_seed => 12345678); 
+```
+```
+INSERT 0 4032
+```
+
+Finally, we can query the aggregate to see various approximate percentiles from different weeks.
```

Comment:
This is nice. I'd also add an example of re-aggregation over the continuous aggregate with the uddsketch(uddsketch)) sort of thing. (If you switch to days rather than weeks, this becomes easier to do, I would be okay with days -> weeks, but we can't go weeks -> months cause time_bucket doesn't support that.